site/docs/tutorial/ios-app.md

```diff
@@ -192,11 +209,14 @@ ios_application(
         "iphone",
         "ipad",
     ],
+    minimum_os_version = "9.0"
     infoplists = [":UrlGet/UrlGet-Info.plist"],
     visibility = ["//visibility:public"],
     deps = [":UrlGetClasses"],
 )
 ```
+**NOTE:** Please update the `minimum_os_version` parameter to the minimum version
```

Comment:
"Please set the value of the `minimum_os_version` attribute to the..." - let's keep our terminology consistent.

Reply:
Fixed